### PR TITLE
cmake: fix mrun to handle cmake build structure

### PR DESCRIPTION
**Pull request by oritwas**
Signed-off-by: Orit Wasserman <owasserm@redhat.com>

### CODE REVIEW

**Comment by oritwas** [comment]
@cbodley I pushed a newer version that handles the conf file too

File: src/mrun [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 [ $# -lt 2 ] && echo "usage: $0 <name> <command> [params...]" && exit 1
 
 root=`dirname $0`
 run_name=$1
 command=$2
+CEPH_BIN=$root
+CEPH_CONF_PATH=$root/run/$run_name
+
+if [ -e CMakeCache.txt ]; then
+    CEPH_BIN=$PWD/src
```
Comment (cbodley) [inline src/mrun]: note that this bit is going to change to `$PWD/bin` once @alimaredia's cmake check work (https://github.com/ceph/ceph/pull/7912) gets merged

File: src/mrun [hunk 2]
```diff
@@ -1,11 +1,18 @@
-#!/bin/sh
+#!/bin/sh -x
```
Comment (cbodley) [inline src/mrun]: helpful for debugging, but we probably don't want to merge this change

**Comment by cbodley** [comment]
thanks @oritwas! for my multisite testing, I had been hacking around this with symlinks. after removing those and testing this pr, there are still a couple issues running ceph/src/test/rgw/test_multi.py. but I'll add fixes for those in a separate pull request - this one looks good :+1:

**Comment by cbodley** [comment]
see https://github.com/ceph/ceph/pull/8246 for the related multisite test fixes

**Comment by oritwas** [comment]
@cbodley removed the -x

**Pull request merged**